<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regular</class>
 <widget class="QDialog" name="regular">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regular Customer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #bfe0f4;</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Drug Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Phone </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Due</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dname">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cost">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_due">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Customer Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Due Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Display/Refresh</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>671</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
